rllib/core/learner/learner_group.py

```diff
@@ -208,14 +219,13 @@ def update_from_batch(
         batch: MultiAgentBatch,
         *,
         async_update: bool = False,
-        reduce_fn: Optional[Callable[[List[Dict[str, Any]]], ResultDict]] = (
-            _reduce_mean_results
-        ),
-        # TODO (sven): Deprecate the following args. They should be extracted from the
-        #  LearnerHyperparameters of those specific algorithms that actually require
-        #  these settings.
+        # TODO (sven): Deprecate the following args. They should be extracted from
+        #  self.config of those specific algorithms that actually require these
+        #  settings.
         minibatch_size: Optional[int] = None,
         num_iters: int = 1,
+        # Already deprecated args.
```

Comment:
Awesome. This reduces a lot of complexity for users. This reduce function was hard to develop w/o having the result dict itself.

Reply:
Yes, totally. It was very messy. Also, it wasn't very clear, where you should reduce, in the Learner (after n SGD iters), in the LearnerGroup (after receiving m Learner results), on the Algo side?